CommentAgent.py

```diff
@@ -228,11 +231,15 @@ def add_pr_messages(self) -> None:
         """
         Adds messages to the GPTWrapper instance related to the pull request.
         """
+        repo_instructions = self.get_custom_instructions()
         pr_deltas = self.GH.get_pr_deltas(self.pr)
         self.GPT.add_message(
             "user",
             GPTsettings.USER_INSTRUCTIONS.format(
-                pr_title=self.pr.title, pr_user=self.pr.user, pr_deltas=pr_deltas
+                repo_instructions=repo_instructions,
+                pr_title=self.pr.title,
+                pr_user=self.pr.user,
+                pr_deltas=pr_deltas,
```

Comment:
The integration of the repository-specific instructions into the user messages is a good enhancement. Ensure to test and verify that the repository instructions are properly being fetched and included.